.gitignore

```diff
@@ -25,4 +25,4 @@ nupkg
 dist
 
 *.doctree.txt
-apiKey.txt
```

Comment:
You sure you meant to get rid of this? Seems like a bad thing to accidentally commit.

Reply:
I had it stored in a text file at first until I realized I just needed to add it to github as a secret, so I don't think it'll be coming back